rhel-edge-84-virt-test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Test script to test RHEL for edge on RHEL 8.4 virtual.
+# Will prepare the env, download upstream test script and run it.
+
+set -uo pipefail
+source /etc/os-release
```

Comment:
Does this script need this?

Reply:
ostree-ng.sh will do the same thing, will remove it.